### PR TITLE
poor quality tasks

### PR DESCRIPTION
**Pull request by GeroiGorodskoyZastroiki**

### CODE REVIEW

**Review by alexthvest — CHANGES_REQUESTED** [review]

File: CourseApp.Tests/CalcTest.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public class CalcTest
+    {
+        [Fact]
+        public void Multiply_100_NOERR()
+        {
+            var calc = new Calc();
+            var a = 5;
+            var b = 20;
+            var exp = 100;
+
```
Comment (jskonst) [inline CourseApp.Tests/CalcTest.cs:14]: let's use inlineData for multiple tests

File: CourseApp/Equation.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        public double Function(double x = 0, double a = 2.0)
+        {
+            return Math.Pow(Math.Log10(a + x), 2) / Math.Pow(a + x, 2);
+        }
+
+        public double[] TaskA(double xn, double xk, double dx)
+        {
+            double[] results = new double[(int)(((xk - xn) / dx) + 1)];
+            int i = 0;
+            while (xn <= xk)
```
Comment (jskonst) [inline CourseApp/Equation.cs:16]: i would prefer to use for loop - it's much saftier

File: CourseApp/Equation.cs [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    public class Equation // вариант 19
+    {
+        public double Function(double x = 0, double a = 2.0)
+        {
+            return Math.Pow(Math.Log10(a + x), 2) / Math.Pow(a + x, 2);
+        }
+
+        public double[] TaskA(double xn, double xk, double dx)
+        {
+            double[] results = new double[(int)(((xk - xn) / dx) + 1)];
```
Comment (jskonst) [inline CourseApp/Equation.cs:14]: Please add test case where xn will be greater than xk

File: CourseApp/Program.cs [hunk 4]
```diff
@@ -6,8 +6,14 @@ public class Program
...
-            Console.WriteLine($"Hello world");
-            Console.ReadLine();
+            // var calc = new Calc();
+            // Console.WriteLine(calc.Multiply(5, 20));
+            // Console.ReadLine();
+
+            // Mouse new_mouse = new Mouse();
+            // Console.WriteLine("Введите возраст мыши:");
+            // new_mouse.Age = Convert.ToInt32(Console.ReadLine());
+            // Console.WriteLine($"Возраст мыши: {new_mouse.Age}.");
```
Comment (jskonst) [inline CourseApp/Program.cs:16]: Please NEVER upload commented code

**Review by jskonst — CHANGES_REQUESTED** [review]

**Pull request closed without merging**